server/types/unknown.go

```diff
@@ -105,7 +105,11 @@ func (u UnknownType) IoInput(ctx *sql.Context, input string) (any, error) {
 
 // IoOutput implements the DoltgresType interface.
 func (u UnknownType) IoOutput(ctx *sql.Context, output any) (string, error) {
-	return output.(string), nil
+	s, ok := output.(string)
+	if !ok {
+		return "", fmt.Errorf("%s: unexpected type %T", u.String(), output)
+	}
+	return s, nil
```

Comment:
This should _always_ be a `string` type, no exceptions. Not a `nil`, nor any other type. If this isn't isn't a `string`, then there's an error somewhere else that needs to be fixed. This should be impossible. We should definitely panic on our assertion if it's false, because something is critically wrong. We should find and fix the setting issue instead.

Reply:
This was panicking and causing the entire suite to not run afterwards, but it seems to have been fixed after merging main. Something to do with prepared statements I think? I'll fix the panic handling if I see it again.